README.md

```diff
@@ -9,7 +9,7 @@ The Evangelism WG tries to bring more people into Node, and add to the experienc
 How to get involved!
 --------------------
 
-There are lots of ways to contribute to [Node.js](https://github.com/nodejs). If you are interested in contribution to Node.js evangelism specifically, feel free to join in the conversations, help with [issues](https://github.com/nodejs/evangelism/issues), and share the [roadmap](https://github.com/nodejs/roadmap/). You don't have to be a member of the Working Group to pitch in!
+There are lots of ways to contribute to [Node.js](https://github.com/nodejs). If you are interested in contribution to Node.js evangelism specifically, feel free to join in the conversations, help with [issues](https://github.com/nodejs/evangelism/issues), and helping others [get started](https://github.com/nodejs/getting-started/). You don't have to be a member of the Working Group to pitch in!
```

Comment:
Maybe don't repeat the word _help_ twice?

> (...) conversations, participate in the issues or help others get started in Node.js

Other than than LGTM.

Reply:
You're totally right - thanks for catching that! Updated.